.coveragerc

```diff
@@ -0,0 +1,4 @@
+[run]
+omit =
+    */site-packages/*
+    */distutils/*
```

Comment:
I have seen a couple of style guide enforcers introduced but wouldn't flake8 not cover most of these issues sufficiently since its a wrapper that checks for a couple of pep specified rules at once.

Reply:
Thanks for the feedback. For code analysis and formatting we using a combination of yapf and pylint. flake8 is also a great tool, but in this case we prioritized alignment with the CLI, which uses the same tools.